src/openrct2-ui/UiContext.cpp

```diff
@@ -119,6 +120,20 @@ class UiContext final : public IUiContext
         {
             SDLException::Throw("SDL_Init(SDL_INIT_VIDEO | SDL_INIT_JOYSTICK)");
         }
+        if (gConfigGeneral.RefreshDPIScaling)
+        {
+            float ddpi, hdpi, vdpi;
+            if (!SDL_GetDisplayDPI(0, &ddpi, &hdpi, &vdpi))
```

Comment:
```suggestion
            if (SDL_GetDisplayDPI(0, &ddpi, &hdpi, &vdpi) == 0)
```
! implies a bool.

Reply:
It returns an error code, 0 in this case means success.

Reply:
I know. What I was saying is that an explicit check for that code makes it clearer it is a code and not a bool.

Reply:
For some reason I thought the suggestion is the actual diff, my bad.